### ISSUE TITLE
Fetching article with complicated HTML markup looks odd

### ISSUE THREAD

**Issue by funwhilelost** [wallabag/wallabag#3402]
### Issue details

When the article [How to Fix Facebook](https://mobile.nytimes.com/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html) is retrieved, some of the HTML markup is not correctly stripped and it causes visual oddities when displayed.

![capture](https://user-images.githubusercontent.com/127476/32393466-cc8e820a-c096-11e7-91ca-17960da66cdc.PNG)

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.0.8
* How did you install wallabag? Docker image
* php version:
* OS: Ubuntu
* type of hosting (shared or dedicated): Dedicated, Docker
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MariaDB

### Steps to reproduce/test case

Import article https://mobile.nytimes.com/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html and the read the article (may need to use the Material theme to repro exactly)

It appears that the original site is overloading the use of the `<i>` element which is presumably allowed through in Wallabag since it's usually used for italicized typography.

The exact markup that appears while reading (for me) is:

```
<article>
  <header>
    <div class="article-body">
      <p class="p-block">Colin Stretch, the general counsel of Facebook, <a href="https://www.nytimes.com/2017/10/31/us/politics/facebook-twitter-google-hearings-congress.html">appeared on Tuesday before senators</a> who are investigating how Russia spread misinformation
        online during the 2016 presidential campaign. Along with Google and Twitter, Facebook has been blamed for helping Russian agents influence the outcome of the election.</p>
      <p class="p-block">But the cloud over Facebook extends far beyond Russia. Critics say the company’s central role in modern communication has undermined the news business, split Americans into partisan echo chambers <a href="https://www.theguardian.com/technology/2017/oct/05/smartphone-addiction-silicon-valley-dystopia">and “hijacked” our minds</a>        with a product designed to keep us addicted to the social network.</p>
      <p class="p-block">Of course, criticism of Facebook and its founder, Mark Zuckerberg, is easy to come by; solutions aren’t as clear. We asked nine technologists, academics, politicians and journalists to propose the steps they would take to improve Facebook — as a
        product, a company or both.</p>
      <p class="p-block">Their responses, edited slightly for length and clarity, are below.</p>
      <h4 class="p-block">Jonathan Albright</h4>
      <div class="related-asset article-image p-block">
        <a href="https://mobile.nytimes.com/images/100000005525916/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link">
          <div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-1JG8/00ZUCKERBERG-slide-1JG8-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-1JG8/00ZUCKERBERG-slide-1JG8-superJumbo.jpg"></i></div>
        </a>
      </div>
      <p class="p-block"><i class="btn icon icon-image"><em>Research director at Columbia University’s Tow Center for Digital Journalism. </em></i></p>
      <p class="p-block"><i class="btn icon icon-image">With a recent Facebook change that algorithmically <a href="http://www.businessinsider.com/facebook-is-pushing-reactions-over-likes-2017-3">prioritizes “reactions” in the news feed</a> over the standard “like,” the company appears to be pushing further into the realm of emotion-centered engagement. As they stand, and especially with algorithmic reinforcement, “reactions” and “likes” are like megaphones for echo chambers and news outrage.</i></p>
      <p
        class="p-block"><i class="btn icon icon-image">The single most important step Facebook — and its subsidiary Instagram, which I view as equally important in terms of countering misinformation, hate speech and propaganda — can take is to abandon the focus on emotional signaling-as-engagement.</i></p>
        <p
          class="p-block"><i class="btn icon icon-image">This is a tough proposition, of course, as billions of users have been trained to do exactly this: “react.”</i></p>
          <p class="p-block"><i class="btn icon icon-image">What if there were a “trust emoji”? Or respect-based emojis? If a palette of six emoji-faced angry-love-sad-haha emotional buttons continues to be the way we engage with one another — and how we respond to the news — then it’s going to be an uphill battle.</i></p>
          <p
            class="p-block"><i class="btn icon icon-image">Negative emotion, click bait and viral outrage are how the platform is “being used to divide.” Given this problem, Facebook needs to help us unite by building new sharing tools based on trust and respect.</i></p>
            <h4
              class="p-block"><i class="btn icon icon-image">Kevin Kelly</i></h4>
              <div class="related-asset article-image p-block"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005525918/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-QZP9/00ZUCKERBERG-slide-QZP9-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-QZP9/00ZUCKERBERG-slide-QZP9-superJumbo.jpg"></i></div>
              </a>
              </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Co-founder of Wired magazine.</em></i></i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image">Facebook should reduce anonymity by requiring real verification of real names for real people, with the aim of having 100 percent of individuals verified.</i></i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image">Companies would need additional levels of verification, and should have a label and scrutiny different from those of people. (Whistle-blowers and dissidents might need to use a different platform.)</i></i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image">Facebook could also offer an optional filter that would keep any post (or share) of an unverified account from showing up. I’d use that filter.</i></i>
    </p>
    <h4 class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"> Ro Khanna</i></i></h4>
    <div class="related-asset article-image p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005526020/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-NJ8S/00ZUCKERBERG-slide-NJ8S-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-NJ8S/00ZUCKERBERG-slide-NJ8S-superJumbo.jpg"></i></div>
    </a>
    </i>
    </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Democrat representing California’s 17th Congressional District, which includes sections of Silicon Valley. </em></i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Ultimately, whether from tech companies or Congress, what people want is more transparency.</i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Facebook should expand on its <a href="https://newsroom.fb.com/news/category/hard-questions/">Hard Questions blog</a> to explain how its news feed algorithms work, how it uses data in targeting and how it makes decisions about third-party verification and removing offensive content. It should make it simple for users to provide feedback and be responsive to their concerns.</i></i>
      </i>
    </p>
    <div class="related-asset article-interactive p-block embedded no-thumb"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://www.nytimes.com/interactive/2015/07/25/technology/100000003820245.embedded.html"><div class="text-container"><p class="image-caption">Interactive Feature | Interested in All Things Tech? The Bits newsletter will keep you updated on the latest from Silicon Valley and the technology industry.</p></div></a></i></i>
      </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">The company also should make its executives readily available to the press, and they should spend time on Capitol Hill to explain their decision-making.</i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Everyone understands that new technology platforms are not perfect, and that bad actors find ways to abuse them. The key is for Facebook to be upfront about technical challenges, open about its mistakes and willing to answer the tough questions honestly. If it does that, it will continue to earn the public’s trust.</i></i>
      </i>
    </p>
    <h4 class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Eli Pariser</i></i></i></h4>
    <div class="related-asset article-image p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005525915/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-A4ON/00ZUCKERBERG-slide-A4ON-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-A4ON/00ZUCKERBERG-slide-A4ON-superJumbo.jpg"></i></div>
    </a>
    </i>
    </i>
    </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Chief executive of Upworthy and author of “The Filter Bubble.”</em></i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Facebook should open itself up for independent research. Right now, Facebook is a black box: It’s very difficult, and in many cases impossible, for researchers to independently look at behavior on the platform. While opening private data to research creates risks, there’s a ton of explicitly public data on the platform that Facebook makes difficult to query at scale. Facebook could also open up many of the tools advertisers currently use for free use by research scientists. It would be a bold move for transparency, and one that would help us understand much better what’s happening on the world’s most important social platform. And it’d be wise to do this before regulators forced them to.</i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">The company should also optimize for “time well spent.” Facebook’s greatest superpower is figuring out how to eat as much of our attention as possible. But as <a href="http://www.tristanharris.com/">Tristan Harris</a> and others have pointed out, that attention often doesn’t yield much — leaving us poorly informed and feeling worse about ourselves. Instead of measuring clicks and likes, what if Facebook optimized for how much value an article or video or game gave us weeks or months afterward? The company could survey the kinds of content we’ve spent the most time on, and ask us which gave us the most and least value, as a way to balance our impulsive present selves with our greater aspirations.</i></i>
      </i>
    </p>
    <h4 class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Kate Losse</i></i></i></h4>
    <div class="related-asset article-image p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005525912/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-NN39/00ZUCKERBERG-slide-NN39-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-NN39/00ZUCKERBERG-slide-NN39-superJumbo.jpg"></i></div>
    </a>
    </i>
    </i>
    </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Early Facebook employee who recounted her time at the company in her book, “The Boy Kings: A Journey Into the Heart of the Social Network.”</em></i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">It would be interesting if Facebook offered a “vintage Facebook” setting that users could toggle to, without News Feed ads and “like” buttons. (Before “likes,” users wrote comments, which made interactions more unique and memorable.)</i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">A “vintage Facebook” setting not only would be less cluttered, it would refocus the experience of using Facebook on the people using it, and their intentions for communication and interaction.</i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Somehow, no matter how “smart” the Facebook algorithms and behind-the-scenes data processing get, the site felt more engaging as a tool for human communication when interaction was primarily driven by what users wanted to do and say, rather than where the algorithms want people to look.</i></i>
      </i>
    </p>
    <h4 class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Alice Marwick</i></i></i></h4>
    <div class="related-asset article-image p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005525911/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-HSFL/00ZUCKERBERG-slide-HSFL-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-HSFL/00ZUCKERBERG-slide-HSFL-superJumbo.jpg"></i></div>
    </a>
    </i>
    </i>
    </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Assistant professor of communication at the University of North Carolina at Chapel Hill.</em></i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Facebook should allow users to sort their news feed chronologically by default on all platforms, rather than rely on an algorithmically sorted News Feed. This would increase the diversity of items in the News Feed, and would make it more likely that users were exposed to people and information that didn’t support their own confirmation bias.</i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">It should also greatly increase the human oversight of ad targeting systems — specifically, oversee algorithmically generated categories (rather than basing them solely on user-inputted interests). Political and interest-based advertising should be under much stricter scrutiny than, say, the advertising of T-shirts or hair products.</i></i>
      </i>
    </p>
    <h4 class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Ellen Pao</i></i></i></h4>
    <div class="related-asset article-image p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005525920/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-H6I4/00ZUCKERBERG-slide-H6I4-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-H6I4/00ZUCKERBERG-slide-H6I4-superJumbo.jpg"></i></div>
    </a>
    </i>
    </i>
    </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Chief diversity and inclusion officer at the Kapor Center for Social Impact and a former chief executive of Reddit.</em></i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Facebook needs to replace its focus on engagement quantity with interaction quality. To really do that means replacing at least half of the leadership team and board with underrepresented people of color who are informed and value diversity and inclusion — and, as my colleague <a href="http://www.kaporcapital.com/who-we-are/freada-kapor-klein/">Freada Kapor Klein</a> suggested, have journalistic principles. At Reddit, I was able to effect positive, lasting change (at least according to <a href="http://comp.social.gatech.edu/papers/cscw18-chand-hate.pdf">this research</a>) to content quality and interaction quality by building a diverse executive team.</i></i>
      </i>
    </p>
    <h4 class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Vivian Schiller</i></i></i></h4>
    <div class="related-asset article-image p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005525914/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-AMHC/00ZUCKERBERG-slide-AMHC-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-AMHC/00ZUCKERBERG-slide-AMHC-superJumbo.jpg"></i></div>
    </a>
    </i>
    </i>
    </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Adviser and former news executive at NPR, NBC News and Twitter.</em></i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">The single most important thing Facebook must do is come clean. Tell us what you know. Tell us what you know but can’t share. Tell us what you don’t know. And tell us what you don’t know that you don’t know. Stop hiding behind bromides like “we are not a media company.” That makes us think you don’t understand you have a serious set of problems that need fixing. Coming clean may not be the only thing, and may not be the main thing, but it’s the first thing.</i></i>
      </i>
    </p>
    <h4 class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Tim Wu</i></i></i></h4>
    <div class="related-asset article-image p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/images/100000005525913/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html" class="article-image-link"><div class="image-container"><i class="btn icon icon-image"><img src="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-JXQF/00ZUCKERBERG-slide-JXQF-articleInline.jpg" class="lightbox-candidate" data-lightbox-image-url="https://cdn1.nyt.com/images/2017/11/01/business/00ZUCKERBERG-slide-JXQF/00ZUCKERBERG-slide-JXQF-superJumbo.jpg"></i></div>
    </a>
    </i>
    </i>
    </i>
    </div>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><em>Professor at Columbia Law School and author of “The Attention Merchants: The Epic Scramble to Get Inside Our Heads.”</em></i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Facebook should become a public benefit corporation. These companies must aim to do something that would aid the public, and board members must also take that public benefit into account when making decisions. Mark Zuckerberg has said that Facebook’s goals are “bringing us closer together” and “building a global community.” Worthy, beautiful goals, but easier said than done when Facebook is also stuck delivering ever-increasing profits and making its platform serve the needs of advertisers.</i></i>
      </i>
    </p>
    <p class="p-block"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">What if Facebook were actually free to do what it says it wants to? What if it didn’t need to devote so much energy to the evil sides of the business, whether catering to filter bubbles, addicting and manipulating users, seizing data, bending over backward for advertisers and destroying competitors? As a nonprofit or public benefit corporation (<a href="https://www.nytimes.com/2015/09/21/technology/kickstarters-altruistic-vision-profits-as-the-means-not-the-mission.html?hpw&amp;rref=business&amp;action=click&amp;pgtype=Homepage&amp;module=well-region&amp;region=bottom-well&amp;WT.nav=bottom-well&amp;_r=0">like Kickstarter</a>), Facebook could be a much better institution. It could shed its “two masters” dilemma, truly pursue its lofty goals and become a firm of which its users and the world could actually be proud.</i></i>
      </i>
    </p>
    </div>
    <footer>
      <div data-view="well" class="well hide story-39-survey-cta"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/2017/10/31/technology/how-to-fix-facebook-we-asked-9-experts.html#" target="_blank" rel="noopener noreferrer"> We're interested in your feedback on this page. <em>Tell us what you think.</em> </a></i></i>
        </i>
      </div>
    </footer>
    <div class="article-related-coverage" data-view="article-related-coverage">
      <h3><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">Related Coverage</i></i></i></h3>
      <ol class="headlines touchstates">
        <li data-id="100000005519765"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/2017/10/29/business/facebook-misinformation-abroad.html" data-link=""><img src="https://cdn1.nyt.com/images/2017/10/28/business/30ROOSE-1/30ROOSE-1-thumbLarge.jpg" class="highlighted-thumb" width="150" height="150">The Shift: Forget Washington. Facebook’s Problems Abroad Are Far More Disturbing. Oct 29, 2017</a></i></i>
          </i>
        </li>
        <li data-id="100000005519702"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/2017/10/27/technology/facebook-fake-content-employees.html" data-link=""><img src="https://cdn1.nyt.com/images/2017/10/28/business/28FACEBOOK1/28FACEBOOK1-thumbLarge.jpg" class="highlighted-thumb" width="150" height="150">At Facebook, Hand-Wringing Over a Fix for Fake Content Oct 27, 2017</a></i></i>
          </i>
        </li>
        <li data-id="100000004612619"><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image"><a href="https://mobile.nytimes.com/2017/09/17/technology/facebook-government-regulations.html" data-link=""><img src="https://cdn1.nyt.com/images/2017/09/21/insider/FB_THUMB/FB_THUMB-thumbLarge.jpg" class="highlighted-thumb" width="150" height="150">Facebook Faces a New World as Officials Rein In a Wild Web Sep 17, 2017</a></i></i>
          </i>
        </li>
      </ol>
    </div><i class="btn icon icon-image"><i class="btn icon icon-image"><i class="btn icon icon-image">
        </i></i>
    </i>
  </header>
</article>
```

**Comment by techexo** [comment]
@funwhilelost Can you reproduce this bug with latest wallabag release (2.3.4, [here](https://github.com/wallabag/docker/releases/tag/2.3.4) for docker). I can't reproduce at home;
If everything is going fine, could you close the issue? Thanks

**Comment by funwhilelost** [comment]
Verified as fixed in 2.3.4, thanks!  🎉
